projects/common/src/lib/textarea/textarea.component.spec.ts

```diff
@@ -16,15 +16,27 @@ describe('TextareaComponent', () => {
   beforeEach(() => {
     fixture = TestBed.createComponent(TextareaComponent);
     component = fixture.componentInstance;
-    fixture.detectChanges();
```

Comment:
Removi esta linha do `beforeEach` e adicionei no início de cada teste. Fiz isso para criar um teste no qual eu pudesse iniciar um componente com o campo value preenchido sem afetar os testes que já existiam.